### DISCUSSION TITLE
Welcome to CookNook Discussions!

### DISCUSSION THREAD

**Discussion by morganog** [morganog/CookNook#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

My name is Morgan Anderson and I'm currently working on building a web app where friends/family can share recipes. This project truly means a lot to me and I hope it can help land me a job! I also hope for anyone contributing that it can help with your job resume as well.
__

Feel free to open a discussion here before posting to issues OR post to issues. I am okay with whatever, but do appreciate the community feel.

**Comment by abshek7** [comment]
Do we have any low level design of application we are building
